frameworks/Java/rapidoid/benchmark_config.json

```diff
@@ -2,11 +2,11 @@
   "framework": "rapidoid",
   "tests": [{
     "default": {
-      "setup_file": "setup",
+      "setup_file": "setup-low-level",
       "json_url": "/json",
       "plaintext_url": "/plaintext",
       "port": 8080,
-      "approach": "Stripped",
+      "approach": "Realistic",
```

Comment:
Why that's not fine?
The code has already been refactored exactly for this purpose.